app/scripts/templates/confirm_account_unlock.mustache

```diff
@@ -0,0 +1,22 @@
+<div id="main-content">
+  <header>
+    <h1 id="fxa-confirm-account-unlock-header">{{#t}}Unlock email sent{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success">
+      {{#t}}Email resent{{/t}}
+    </div>
+
+    <div class="graphic graphic-mail">{{#t}}Email Sent{{/t}}</div>
+
+    <p class="verification-email-message">{{#t}}For security reasons your account was put on lockdown. To perform account-level activities, please re-verify your account by clicking the verification link sent to %(email)s{{/t}}</p>
+
+    <div class="links">
+      <a id="back" class="left" href="#">{{#t}}Back{{/t}}</a>
+      <a id="resend" class="right" href="#">{{#t}}Resend email{{/t}}</a>
```

Comment:
> However note that we currently do not send email at the time the account is locked. So the first time the user sees this page, we will not have actually sent them any email and they will have to click "resend email" to get out of this state.


Yup, and nope. When the account is locked, we display an error message that allows the user to click a link to unlock their account:


![screen shot 2015-03-09 at 09 56 14](https://cloud.githubusercontent.com/assets/848085/6552896/b20e4d30-c642-11e4-9d3c-7813ddd999b7.png)

If the user clicks the link, an email is sent and this screen is displayed. This is a long winded way of saying, "the resend button is applicable here."